apps/src/templates/HeadingBanner.jsx

```diff
@@ -5,39 +5,77 @@
 
 import React from 'react';
 import color from "../util/color";
+import ProgressButton from './progress/ProgressButton';
+import i18n from "@cdo/locale";
 
 const styles = {
   bannerHeading: {
     fontFamily: '"Gotham 7r", sans-serif',
     color: color.white,
     fontSize: 32,
-    marginTop: 40,
     marginBottom: 10,
     lineHeight: '40px'
   },
   bannerSubHeading: {
     fontFamily: '"Gotham 4r", sans-serif',
     color: color.white,
-    fontSize: 13,
-    marginBottom: 145
+    fontSize: 16,
+    lineHeight: '32px',
+    marginBottom: 10
+  },
+  bannerDescription: {
+    fontFamily: '"Gotham 4r", sans-serif',
+    color: color.white,
+    fontSize: 16,
+    width: '600px',
+    marginBottom: 20,
+  },
+  spacer: {
+    marginBottom: '50px'
+  },
+  bigSpacer: {
+    marginBottom: '150px'
   }
 };
 
 const HeadingBanner = React.createClass({
   propTypes: {
+    extended: React.PropTypes.bool,
     headingText: React.PropTypes.string,
-    subHeadingText: React.PropTypes.string
+    subHeadingText: React.PropTypes.string,
+    description: React.PropTypes.string,
   },
 
   render() {
+    const {extended, headingText, subHeadingText, description} = this.props;
+
     return (
       <div>
+        {!extended && (
+          <div style={styles.spacer}/>
+        )}
         <div style={styles.bannerHeading}>
-          {this.props.headingText || <span>&nbsp;</span>}
+          {headingText || <span>&nbsp;</span>}
         </div>
         <div style={styles.bannerSubHeading}>
-          {this.props.subHeadingText || <span>&nbsp;</span>}
+          {subHeadingText || <span>&nbsp;</span>}
         </div>
+        {extended ? (
```

Comment:
I would also encourage the use of the syntax
```jsx
{extended && <Thing1/>}
{!extended && <Thing2/>}
```
instead of `{extended ? <Thing1/> : <Thing2/>}` as I find the ternary operator difficult to parse in jsx